### PR TITLE
Add metabot raw data toggle

### PR DESCRIPTION
**Pull request by ranquild**
Epic https://github.com/metabase/metabase/issues/29859

Adds viz/raw data toggle to the metabot page. Currently can be only tested via Cypress.

Screenshot 2023-04-06 at 18 28 39

- - -
This change is [Reviewable](https://reviewable.io/reviews/metabase/metabase/29844)

### CODE REVIEW

File: frontend/src/metabase/metabot/actions.ts [hunk 1]
```diff
@@ -87,6 +91,11 @@ export const RUN_QUESTION_QUERY_REJECTED =
 export const runQuestionQuery =
   () => async (dispatch: Dispatch, getState: GetState) => {
     try {
+      const question = getQuestion(getState());
+      if (question) {
+        await dispatch(updateQuestion(getRawTableQuestion(question)));
```
Comment (ranquild) [inline frontend/src/metabase/metabot/actions.ts:96]: We can't use the same display/settings from the prompt question, so we have to set a raw question when running the edited query manually

File: frontend/src/metabase/metabot/components/MetabotFeedback/MetabotFeedback.tsx [hunk 2]
```diff
@@ -37,22 +33,6 @@ const MetabotFeedback = ({
   feedbackType,
   onSubmitFeedback,
   onRetry,
-}: MetabotFeedbackProps) => {
-  return (
-    <MetabotFeedbackRoot>
```
Comment (ranquild) [inline frontend/src/metabase/metabot/components/MetabotFeedback/MetabotFeedback.tsx:42]: Removes the wrapper with a padding

File: frontend/src/metabase/metabot/components/MetabotQueryBuilder/MetabotQueryBuilder.tsx [hunk 3]
```diff
@@ -96,4 +102,17 @@ const QueryErrorState = ({ queryError }: QueryErrorStateProps) => {
...
 
+interface QueryFooterProps {
+  isVisualized: boolean;
+}
+
+const QueryFooter = ({ isVisualized }: QueryFooterProps) => {
+  return (
+    <QueryFooterRoot>
+      <MetabotFeedback />
+      {isVisualized && <MetabotDisplayToggle />}
```
Comment (ranquild) [inline frontend/src/metabase/metabot/components/MetabotQueryBuilder/MetabotQueryBuilder.tsx:113]: As in the qb, don't show the table if the display is table

File: frontend/src/metabase/query_builder/components/view/QuestionDisplayToggle/QuestionDisplayToggle.styled.tsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import styled from "@emotion/styled";
+import { color, darken } from "metabase/lib/colors";
+
+export const Well = styled.div`
```
Comment (ranquild) [inline frontend/src/metabase/query_builder/components/view/QuestionDisplayToggle/QuestionDisplayToggle.styled.tsx:4]: Moved from `ViewFooter`

File: frontend/src/metabase/query_builder/components/view/QuestionDisplayToggle/QuestionDisplayToggle.tsx [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+import { Well, ToggleIcon } from "./QuestionDisplayToggle.styled";
+
+interface QuestionDisplayToggleProps {
+  className?: string;
+  question: Question;
+  isShowingRawTable: boolean;
+  onToggleRawTable: (isShowingRawTable: boolean) => void;
+}
+
+const QuestionDisplayToggle = ({
```
Comment (ranquild) [inline frontend/src/metabase/query_builder/components/view/QuestionDisplayToggle/QuestionDisplayToggle.tsx:15]: Moved from `ViewFooter`

**Review by alxnddr — APPROVED** [review]
Looks good!

When I rerun a query from the query editor the toggle disappears. I think we should preserve it because one of the scenarios is when users get an invalid SQL and want to tweak it

**Pull request merged**